Tests/CryptoTests/Authenticated Encryption/AES-GCM-Runner.swift

```diff
@@ -70,8 +70,8 @@ class AESGCMTests: XCTestCase {
         let recoveredPlaintext = try! AES.GCM.open(ciphertext, using: key, authenticating: Data())
```

Comment:
I think we should change the signature if this function to be throwing, and remove the `!` from `try!`. As seen in many of the `AESGCMTests` tests.

Reply:
+1 here's my favorite utility for that, since just marking a test function as `throws` is kind of lackluster.

```swift
/// Takes a throwing closure. Returns the result if the closure succeeds.
/// If the closure throws, this method registers an XCTest failure and rethrows the error.
/// - Note: closure comes last so that trailing closure syntax works.
func orFail<T>(file: StaticString = #file, line: UInt = #line, _ closure: () throws -> T) throws -> T {
    do {
        return try closure()
    }
    catch {
        XCTFail("Function threw error: \(error)", file: file, line: line)
        throw error
    }
}
```

Reply:
agreed! I use stuff like that as well, good idea! :D